README.md

```diff
@@ -24,7 +24,7 @@ For all projects with `application` applied, each has a task `showServiceChangeS
 By running:
 
 ```shell
-./gradlew showServiceChangeStatus \
+./gradlew showBuildTargetsForChange \
```

Comment:
this was just incorrect, the task name has always been `showBuildTargetsForChange`